strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/storage/repository/RepositoryPolicyEnum.java

```diff
@@ -25,9 +41,22 @@ public String getPolicy()
         return policy;
     }
 
-    public void setPolicy(String policy)
+    public boolean acceptsReleases()
     {
-        this.policy = policy;
+        return true;
```

Comment:
Defaulting to `true` here (and below) is not a very good idea, to be honest. Most repositories will have either a `snapshot`, or `release` policy. Usually, (statistically speaking), these are much more than the `group` repositories. Hence, defaulting to `false` would be more reasonable. Although, this, in a way, is cosmetic.